src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -596,6 +598,21 @@ protected void deletePaymentAccountsOfGivenOrganisation(PbaRequest deletePbaRequ
         paymentAccountService.deletePaymentsOfOrganisation(deletePbaRequest, existingOrganisation);
     }
 
+    protected ResponseEntity<Object> fetchUsersForRefresh(String since, String userId, Integer page, Integer size) {
```

Comment:
I wonder if this better belongs in the ProfessionalUserService. Its not used by any other controller is it?

Reply:
All the controllers extend the `SuperController`. The service layer is not injected in any of the controllers other than the `SuperController`. Also there are other methods only used by 1 controller in the `SuperController` e.g. `retrieveAllOrganisationOrById`, `retrievePaymentAccountByUserEmail`, `retrieveAllOrganisationsByStatus` etc.. - I just went with the current convention.

Reply:
after having a second thought. i think you're right - since it's only used by 1 controller. see commit [97b711e](https://github.com/hmcts/rd-professional-api/pull/1529/commits/97b711e75534414f5e69c726296b7ee32fc59ce5) for refactor